examples/model_compress/quantization/QAT_torch_quantizer.py

```diff
@@ -87,7 +68,7 @@ def main():
     quantizer.compress()
 
     model.to(device)
-    for epoch in range(40):
+    for epoch in range(1):
```

Comment:
why `1`?

Reply:
This is modified for fast validation. Changed back to 40.